metron-platform/Performance-tuning-guide.md

```diff
@@ -219,38 +219,49 @@ And we ran our bro parser topology with the following options. We did not need t
 though you could certainly do so if necessary. Notice that we only needed 1 worker.
 
 ```
-/usr/metron/0.4.0/bin/start_parser_topology.sh -k $BROKERLIST -z $ZOOKEEPER -s bro -ksp SASL_PLAINTEXT
-    -ot enrichments
+/usr/metron/0.4.2/bin/start_parser_topology.sh \
     -e ~metron/.storm/storm-bro.config \
     -esc ~/.storm/spout-bro.config \
-    -sp 24 \
-    -snt 24 \
+    -k $BROKERLIST \
+    -ksp SASL_PLAINTEXT \
```

Comment:
This will only work in a Kerberized environment, no?  Should that be called out?